docs/source/conf.py

```diff
@@ -39,10 +41,10 @@
 
 # The default replacements for |version| and |release|, also used in various
 # other places throughout the built documents.
-d = {}
-with open(os.path.join('..', '..', 'pyface', '_version.py')) as fp:
-    exec (fp.read(), d)
-version = release = d['full_version']
+version_py = os.path.join('..', '..', 'pyface', '_version.py')
+version_content = runpy.run_path(version_py)
+version = ".".join(version_content["version"].split(".", 2)[:2])
+release = version
```

Comment:
This is somewhat a drive-by. The new logic here is consistent with the one in TraitsUI. However Pyface documentation has not used `|version|` anywhere.